src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -505,17 +594,21 @@ const buildIndividualStat = (
   const {
     primaryValuePostfix = '',
     label,
-    primaryUnit
+    primaryUnit,
+    preLabel,
+    helpText
   } = statsFormattingOptions[section][primaryKey] as StatsFormattingOption;
 
   if (typeof primaryValue === 'number') {
     primaryValue = getCommaSeparatedNumber(primaryValue) + primaryValuePostfix;
   }
 
   return {
+    preLabel,
     label: label || primaryKey,
     primaryValue,
-    primaryUnit: primaryUnit
+    primaryUnit: primaryUnit,
```

Comment:
Since both the key and value are `primaryUnit` we can avoid specifying the value here